blocks/procedures.js

```diff
@@ -126,7 +126,7 @@ Blockly.Blocks['procedures_defnoreturn'] = {
   /**
    * Create XML to represent the argument inputs.
    * @param {=boolean} opt_paramIds If true include the IDs of the parameter
-   *     quarks.  Used by Blockly.Procedures.mutateCallers for reconnection.
+   *    quarks.  Used by Blockly.Procedures.mutateCallers for reconnection.
```

Comment:
Why three indents?
First example I looked at in Closure uses 4:
https://github.com/google/closure-library/blob/master/closure/goog/array/array.js#L89